src/plugins/elasticsearch/index.js

```diff
@@ -24,7 +24,7 @@ module.exports = function (kibana) {
           key: Joi.string()
         }).default(),
         apiVersion: Joi.string().default('2.0'),
-        minimumVerison: Joi.string().default('2.0.0')
+        minimumVerison: Joi.string().default('2.1.0')
```

Comment:
Uh, I just noticed that this property is misspelled.  It's possible this was never doing what we expected it to do.